lib/fog/kubevirt/compute.rb

```diff
@@ -7,6 +7,8 @@ class Kubevirt < Fog::Service
       recognizes :kubevirt_host, :kubevirt_port
 
       model_path 'fog/kubevirt/models/compute'
+      model      :livevm
+      collection :livevms
       model      :offlinevm
       collection :offlinevms
       model      :template
```

Comment:
shouldn't we declare also ```tempates``` collection so the user will be able to list them, or would you like it to be added out of the scope of this PR ?

Since we do support list_templates, I think we should introduce at the same PR the 'templates' collection.

Reply:
correct, will add

Reply:
Checked it one more time and I see that it is there line 13.